### DISCUSSION TITLE
How to use AutoComplete custom option with typescript

### DISCUSSION THREAD

**Discussion by Bowen7** [zeit-ui/react#495] [Q&A] [answered]
When using AutoComplete custom option with typescript, I got a type problem: Type 'Element[]' is not assignable to type 'AutoCompleteOptions'.

[The minimal example](https://codesandbox.io/s/hidden-lake-8swbt?file=/src/App.tsx)

**Comment by unix** [comment, marked as answer]
This is a type error, thank you for pointing it out. We have just out patch version(`2.1.4`), you can update Geist UI (CDN sync may take some time) and try again.
